app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for peoples in people:
+        Person(peoples["name"], peoples["age"])
+    for peoples in people:
```

Comment:
Similar to line 11, the variable name `peoples` is misleading. Consider renaming it to `person_data` or something similar.